src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallSummaryGeneratorTest.kt

```diff
@@ -153,7 +155,9 @@ class RecallSummaryGeneratorTest {
         null,
         PrisonName("Current Prison"),
         PrisonName("Last Release Prison"),
-        true
+        true,
+        arrestIssues = false,
```

Comment:
I'm not saying it matters now but won't this be null rather than false, at least potentially, when inCustody = true?